### PR TITLE
Submodules changes

### PR DESCRIPTION
**Pull request by lumburovskalina**
PRs:
- https://github.com/code-kern-ai/admin-dashboard/pull/7
- https://github.com/code-kern-ai/workflow-ui/pull/19
- https://github.com/code-kern-ai/gates-ui/pull/11
- https://github.com/code-kern-ai/refinery-ui/pull/142

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -5,3 +5,8 @@ UI for [refinery](https://github.com/code-kern-ai/refinery). Used to interact wi
 
 
 If you like what we're working on, please leave a ⭐ for [refinery](https://github.com/code-kern-ai/refinery)!
+
+Note:
+The javascript function can be replaced with shared functions from the submodules. The current version of refinery has some issues with the submodules and they are not added. In order to include the submodules the following command should be used:
```
Comment (JWittmeyer) [inline README.md]: the current version of refinery-ui (angular)

**Comment by JWittmeyer** [comment]
i thought we wanted to add the function as a blank copy of the current state of the javascript submodule?

- [ ] discussed
- [x] resovled

Lina: With blank copy is meant as manually created folder submodules/javascript-functions? --> yes
Lina: Yes it makes total sense, I included the submodules but I didn't add them as part of the drone build

File: .gitmodules [hunk 2]
```diff
@@ -0,0 +1,3 @@
+[submodule "submodules/javascript-functions"]
+	path = submodules/javascript-functions
+	url = git@github.com:code-kern-ai/submodule-javascript-functions.git
```
Comment (JWittmeyer) [inline .gitmodules]: not what was meant with blank copy. I meant the files in the normal file structure added without the submodule integration via git. just a copy of the files so they are accessible

Comment (lumburovskalina) [reply .gitmodules]: Okay :) but I shouldn't actually replaced those functions in the components right?

**Comment by JWittmeyer** [comment]
Conflicts because of the mege of lf exec env
image

- [x] resolved

**Review by JWittmeyer — APPROVED** [review]

**Pull request merged**